src/service/plugins/battery.js

```diff
@@ -257,6 +257,64 @@ var Plugin = GObject.registerClass({
         });
     }
 
+    /**
+     * Notify the user the remote battery is at 60% charge.
+     */
+    _sixtyBatteryNotification() {
+        if (!this.settings.get_boolean('sixty-battery-notification'))
```

Comment:
Your tests are failing because this setting is `false` by default. You can fix the tests by toggling the setting in the unit test before sending the packet.

Reply:
Thank you. I have never worked with CI tests before, or with a public git repo for that matter. I will try to enable them in my fork to keep the upstream repo as clean as possible. I would greatly appreciate any corrections or tips on how to keep the clutter down while testing.

I am working on creating a switch to enable a custom level alert, and then a spinbutton to determine what that level is. I was going to leave this off by default, as I imagine most people fully charge their phones all the time.

Reply:
You should be able to run the tests locally with `meson test -C _build` (see [meson docs](https://mesonbuild.com/Unit-tests.html#testing-tool)). Probably we should add directions for running tests in the `CONTRIBUTING.md` page...

Reply:
I could not get local testing to fully work, ESLint kept telling me that the path needed to be a relative path instead of an absolute one.

The other tests ran without a problem though, which was a great help!